homeassistant/components/spotify/browse_media.py

```diff
@@ -171,9 +174,15 @@ async def async_browse_media(
 
     # Check for config entry specifier, and extract Spotify URI
     parsed_url = yarl.URL(media_content_id)
-    if (info := hass.data[DOMAIN].get(parsed_url.host)) is None:
+
+    if (
+        parsed_url.host is None
+        or (entry := hass.config_entries.async_get_entry(parsed_url.host)) is None
+        or not isinstance(entry.runtime_data, HomeAssistantSpotifyData)
```

Comment:
What would it else be?

Reply:
this is more or less to make mypy happy, since `entry.runtime_data` is of generic type. maybe replacing this with an `assert isinstance(entry.runtime_data, HomeAssistantSpotifyData)` below 🤔

Reply:
The current route is correct, we have no better route at this point.

../Frenck